tests/system/action/option/test_update.py

```
@@ -111,7 +125,7 @@ def test_update_invaid_keys(self) -> None:
             "option.update",
             {
                 "id": 57,
-                "A": "1.000000",
+                "A": "0.000000",
```

Why is this test changed?

---

Finally doesn't matter, but I wanted to demonstrate, that 0 votes should raise the exception

---

Okay.